File: 2022/01/0101.py
f = open("01-input.txt", "r")

max = 0;
counter = 0;
for x in f:
    if x.strip() == "":
        if counter > max:
            max = counter;
        counter = 0;
        continue;
    
    counter += int(x);
print(max);

f.close();

File: 2022/01/0101.kt
import java.io.File

fun main() {
    var max: Int = 0
    var counter: Int = 0
    var cont: Boolean = false
    
    File("input.txt").forEachLine { it ->
        if (it.isEmpty()) {
            if (counter > max) {
                max = counter
            }
            counter = 0
            cont = true
        }
        if (!cont) {
            counter += it.toInt()
        }
        cont = false
    }
    println(max)
}